### PR TITLE
Add support for '!discord bridge guildId/channelId' syntax

### PR DESCRIPTION
**Pull request by swedneck**
Solves #273

### CODE REVIEW

File: src/matrixroomhandler.ts [hunk 1]
```diff
@@ -271,8 +271,25 @@ export class MatrixRoomHandler {
...
+            if (args.length === AMOUNT_OF_IDS_DISCORD_IDENTIFIES_ROOMS_BY) { // "x y" syntax
+                guildId = args[0];
+                channelId = args[1];
+            } else if (args.length === 1 && args[0].includes("/")) { // "x/y" syntax
+                const split = args[0].split("/");
+                guildId = split[0];
+                channelId = split[1];
+            } else {
+                return this.bridge.getIntent().sendMessage(event.room_id, {
+                    body: "Invalid syntax",
```
Comment (Sorunome) [inline src/matrixroomhandler.ts]: Maybe a bit more descriptive output, like explaining what the valid syntax is?

Comment (swedneck) [reply src/matrixroomhandler.ts]: How do you handle formatted body in the tests?

Comment (swedneck) [reply src/matrixroomhandler.ts]: I've changed the error message to "Invalid syntax: See `!discord help`"

File: src/matrixroomhandler.ts [hunk 2]
```diff
@@ -271,8 +271,25 @@ export class MatrixRoomHandler {
...
+            if (args.length === AMOUNT_OF_IDS_DISCORD_IDENTIFIES_ROOMS_BY) { // "x y" syntax
+                guildId = args[0];
+                channelId = args[1];
+            } else if (args.length === 1 && args[0].includes("/")) { // "x/y" syntax
+                const split = args[0].split("/");
+                guildId = split[0];
+                channelId = split[1];
+            } else {
+                return this.bridge.getIntent().sendMessage(event.room_id, {
+                    formatted_body: "Invalid syntax: See <code>!discord help</code>",
```
Comment (Sorunome) [inline src/matrixroomhandler.ts]: you still need a `body` even if you have a `formatted_body`, else that is invalid matrix stuff

Comment (Sorunome) [reply src/matrixroomhandler.ts]: `formated_body` is basically just there to make things look pretty in clients that support it, and `body` is the fallback that always has to be htere

Comment (swedneck) [reply src/matrixroomhandler.ts]: So `body` should be "Invalid syntax: See !discord help"?

Comment (Sorunome) [reply src/matrixroomhandler.ts]: preferably with backticks:
```
"Invalid syntax: See `!discord help`"
```

Comment (swedneck) [reply src/matrixroomhandler.ts]: Ah, so markdown?

Comment (Sorunome) [reply src/matrixroomhandler.ts]: yeah, that seems to be the thing most use as fallback

**Comment by Sorunome** [comment]
looks good, thank you!

**Pull request merged**